ci/run_autoscaling_osp18.yml

```diff
@@ -62,3 +85,14 @@
               oc patch  openstackversions controlplane  --type json -p='[{"op": "replace", "path": "/spec/customContainerImages", "value": {} }]'
           when: "{{ patch_openstackversions | bool }}"
 
+        - name: Redeploy the dataplane
```

Comment:
Why is the dataplane being redeployed twice?

Reply:
It's redeployed again to revert the changes done by the first redeployment. So after the first time we have master ceilometer containers. After the second time we have antelope containers again.

Reply:
right

Reply:
Dataplane is updated only as a reaction to a new OpenstackDataplaneDeployment CR. That's why there are tasks to copy the current osdpd into a file, delete the osdpd and apply it again from the file.